examples/fishjam-chat/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
-    "eas-build-pre-install": "yarn workspace fishjam-mobile-sdk install && yarn workspace fishjam-mobile-sdk build",
+    "eas-build-pre-install": "corepack enable && yarn workspace fishjam-mobile-sdk install && yarn workspace fishjam-mobile-sdk build",
```

Comment:
(might be done as separate task)

WDYT about having separate script for these commands? 🤔

Reply:
It must be like that, this is the only script that eas runs before building.